OpenShift/helm/nexus-repository-manager/values.yaml

```diff
@@ -9,8 +9,8 @@ deploymentStrategy: {}
   # type: RollingUpdate
 
 nexus:
-  imageName: registry.connect.redhat.com/sonatype/nexus-repository-manager
-  imageTag: 3.21.1-01-ubi-23
+  imageName: sonatype/nexus3
```

Comment:
same concern as earlier - I believe this should remain using RH as the default image source

Reply:
same reply, we should use our stock images here in a helm chart and John can switch to the RHCC for the operator